web/forms.py

```
@@ -1869,3 +1872,58 @@ class StudyGroupForm(forms.ModelForm):
+ALLOWED_TAGS = ["b", "i", "strong", "em", "ul", "ol", "li", "p", "a"]
+ALLOWED_ATTRIBUTES = {
+    "a": ["href", "title", "target"],
+}
+
+
+class VideoRequestForm(forms.ModelForm):
+    """Form for users to request educational videos on specific topics, with XSS protection."""
+
+    class Meta:
+        model = VideoRequest
+        fields = ["title", "description", "category"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)